main.py

```diff
@@ -73,7 +73,32 @@
 
     await bot.process_commands(message)
 
+#start CuckTim Section
+#replaces "Cuck" with Rotating Light Emoji, meant to piss off Tim
+@bot.event
+async def on_message(message):
```

Comment:
`on_message` is a function that is built into the `discord.py` library and is left open as a template method and since python is duck typed, you can't do method overloading.

In simpler terms, you can't just copy paste the method, cause python won't know which one to use, you have to combine it all in a single `on_message` method (which already exists)